modules/auxiliary/server/capture/ldap.rb

```diff
@@ -59,328 +53,22 @@ def initialize(info = {})
-      })
-        print_status('NLMv1 Hash correspond to an empty password, ignoring ... ')
-        return
-      end
-      if lm_hash == nt_hash || lm_hash == '' || lm_hash =~ /^0*$/
-        lm_hash_message = 'Disabled'
-      elsif NTLM_CRYPT.is_hash_from_empty_pwd?({
-        hash: [lm_hash].pack('H*'),
-        srv_challenge: @challenge,
-        ntlm_ver: NTLM_CONST::NTLM_V1_RESPONSE,
-        type: 'lm'
-      })
-        lm_hash_message = 'Disabled (from empty password)'
+  def primer
+    service.processed_pdu_handler(Net::LDAP::PDU::BindRequest) do |processed_data|
+      if processed_data[:error_msg]
+        print_error(processed_data[:error_msg])
       else
-        lm_hash_message = lm_hash
-      end
-
-      capture_message =
-        "#{captured_time}\nLDAP Login Attempt(NTLMv1 Response) => From #{host} \n" \
-        "USER: #{user} \tLMHASH:#{lm_hash_message || '<NULL>'} \tNTHASH:#{nt_hash || '<NULL>'}\n"
-      capture_message += " Domain:#{domain}" if domain
-      hash = [
-        lm_hash || '0' * 48,
-        nt_hash || '0' * 48
-      ].join(':').gsub(/\n/, '\\n')
-      arg[:private] = hash
-    when NTLM_CONST::NTLM_V2_RESPONSE
-      if NTLM_CRYPT.is_hash_from_empty_pwd?({
-        hash: [nt_hash].pack('H*'),
-        srv_challenge: @challenge,
-        cli_challenge: [nt_cli_challenge].pack('H*'),
-        user: user,
-        domain: domain,
-        ntlm_ver: NTLM_CONST::NTLM_V2_RESPONSE,
-        type: 'ntlm'
-      })
-        print_status('NTLMv2 Hash correspond to an empty password, ignoring ... ')
-        return
-      end
-      if (lm_hash == '0' * 32) && (lm_cli_challenge == '0' * 16)
-        lm_hash_message = 'Disabled'
-      elsif NTLM_CRYPT.is_hash_from_empty_pwd?({
-        hash: [lm_hash].pack('H*'),
-        srv_challenge: @challenge,
-        cli_challenge: [lm_cli_challenge].pack('H*'),
-        user: user,
-        domain: domain,
-        ntlm_ver: NTLM_CONST::NTLM_V2_RESPONSE,
-        type: 'lm'
-      })
-        lm_hash_message = 'Disabled (from empty password)'
-      else
-        lm_hash_message = lm_hash
-      end
-
-      capture_message =
-        "#{captured_time}\nLDAP Login Attempt(NTLMv2 Response) => From #{host} \n" \
-        "USER: #{user} \tLMHASH:#{lm_hash_message || '<NULL>'}\tNTHASH:#{nt_hash || '<NULL>'} "
-      capture_message += " DOMAIN: #{domain}" if domain
-      hash = [
-        lm_hash || '0' * 32,
-        nt_hash || '0' * 32
-      ].join(':').gsub(/\n/, '\\n')
-      arg[:private] = hash
-    when NTLM_CONST::NTLM_2_SESSION_RESPONSE
-      if NTLM_CRYPT.is_hash_from_empty_pwd?({
-        hash: [nt_hash].pack('H*'),
-        srv_challenge: @challenge,
-        cli_challenge: [lm_hash].pack('H*')[0, 8],
-        ntlm_ver: NTLM_CONST::NTLM_2_SESSION_RESPONSE,
-        type: 'ntlm'
-      })
-        print_status('NTLM2_session Hash correspond to an empty password, ignoring ... ')
-        return
+        print_good(processed_data[:result_message])
```

Comment:
Testing shows the output here changes format from previous iterations:
```
[+] LDAP Login Attempt => From:127.0.0.1:56784
 Username:test
 password:smoking
 Domain:cqure.net
```

Consider adjusting how this is displayed, ofter module output is parsed by user for one reason or another and the multiline output here makes it more complex to extract the important lines from the output.